### ISSUE TITLE
A bug with blur when scrolling

### ISSUE THREAD

**Issue by niklissad** [monterail/vue-multiselect#100]
**Steps to reproduce:** 
1. Start typing in search field
2. Click to scrollbar for scroll

**Expected result:**
Scroll search results

**Actual result:**
Empty search field

**Comment by shentao** [comment]
Will investigate it. Thanks! 
Congrats on the #100 issue/pr created! :)

**Comment by niklissad** [comment]
Thanks )
My solution is to check relatedTarget of event that call deactivate
Smth like
```
deactivate (e) {
      ...
      if (e && e.relatedTarget !== null) {
        this.$els.search.focus()
        return
      }
      ...
}
```

**Comment by shentao** [comment]
Got it fixed! Thanks :)
The updated release should be available ~Wednesday, as I want to fix some other issues before.

**Comment by shentao** [comment]
Finally published and available in v1.1.3!

**Comment by rudivdme** [comment]
Had this same issue with v2. It can be seen on the docs page for the "Asynchronous select" example. Type to search for a country and get multiple results, go to grab the scrollbar, and the search field is cleared, and the results updated. 

Oh wait I found an undocumented prop preserveSearch which resolved it. Thanks!

`:preserve-search="true"`
